### PR TITLE
add bold text

### PR DESCRIPTION
**Pull request by khroponiuk**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,3 @@
 # testing
 testing
+<b>Bold</b>
```
Comment (akhroponiuk) [inline README.md]: Add <i>italic</i>

Comment (akhroponiuk) [inline README.md]: Is it OK?

Comment (akhroponiuk) [inline README.md]: Ok

Comment (akhroponiuk) [inline README.md]: I think yes

Comment (khroponiuk) [inline README.md]: Ok

**Pull request merged**